fastlane/lib/fastlane/actions/git_default_remote_branch.rb

```diff
@@ -0,0 +1,57 @@
+module Fastlane
+  module Actions
+    class GitDefaultRemoteBranchAction < Action
+      def self.run(params)
+        Actions.git_default_remote_branch_name(params[:remote_name])
```

Comment:
ditto ⬆️
I think, this can become just `Actions.git_remote_branch_name(params[:remote_name])`

Reply:
Done 👍